src/util/dev/builder.ts

```diff
@@ -390,6 +390,10 @@ export async function getBuildMatches(
       src = src.substring(1);
     }
 
+    // We need to escape brackets since `glob` will
+    // try to find a group otherwise
+    src = src.replace(/(\[|\])/g, '[$1]');
```

Comment:
Just to be clear, `api/[foo]/bar.js` becomes `api/[[]foo[]]/bar.js`?

Reply:
@TooTallNate exactly, it'll match a group of this specific character, because escaping with `\` didn't work on Windows

Reply:
Interesting! Thanks for the explanation.